boostedblob/azure_auth.py

```diff
@@ -25,6 +25,10 @@
 
 
 def load_credentials() -> Dict[str, Any]:
+    # When USE_AZURE_IDENTITY=1, boostedblob will use the azure-identity package to retrieve AAD access tokens
+    if os.getenv("USE_AZURE_IDENTITY", "0") == "1":
```

Comment:
Can we name the env var `AZURE_USE_IDENTITY`? (I prefer env vars with meaningful prefix)